.github/workflows/checks.yml

```
@@ -73,7 +73,7 @@ jobs:
           - python-version: "3.8"  # avoid uploading coverage for full matrix
             use_coverage: true
           - python-version: "3.8"  # this is to avoid installing optional dependencies in all environments
-            optional-dependencies: tensorflow sympy torch
+            optional-dependencies: tensorflow sympy torch sklearn
```

Here I believe `optional-dependencies` expects a package distribution name (i.e. what you `pip install`), rather than a package name (i.e. what you `import`), therefore I think it should be:

```suggestion
            optional-dependencies: tensorflow sympy torch scikit-learn
```

---

Thanks @lbianchi-lbl, I will address your comments and check that everything runs smoothly.